plugins/Actions/API.php

```diff
@@ -472,17 +481,40 @@ protected function doFilterPageDatatableSearch($callBackParameters, $table, $sea
      *
      * @param DataTable|DataTable\Simple|DataTable\Map $dataTable
      * @param bool $isPageTitleType Whether we are handling page title or regular URL
+     * @param bool $includeGoals Whether goals columns should be included in the result
      */
-    private function filterActionsDataTable($dataTable, $isPageTitleType)
+    private function filterActionsDataTable($dataTable, $isPageTitleType, $includeGoals = false)
```

Comment:
Agreed it would be more consistent with the rest of the goals implementation. I'll change `includeGoals` to default to true. It was disabled by default because it could potentially add hundreds of columns to the reports, which might be problematic for existing third party API integrations. So maybe we'd need document this as a potential breaking change for the release. There will be quite a few additional tests to adjust too.